### PR TITLE
Feat/list request

### PR DESCRIPTION
**Pull request by rayfajars**
## OVERVIEW

1. add filter list table
2. add static data type request, division , and type item
3. add styling detail, verifikasi , pengembalian request

## PREVIEW

image

title: add status and styling detail
project: Digiteam Inventaris Platform
participants: @rayfajars @Ibwedagama @yoslie

### CODE REVIEW

File: src/components/layouts/ActionTable.vue [hunk 1]
```diff
@@ -15,15 +15,29 @@ export default {
...
     },
+    toggleDropdown() {
+      this.show = !this.show;
+    },
+    close(e) {
+      if (!this.$el.contains(e.target)) {
+        this.show = false;
+      }
+    },
   },
```
[the remark this answers is not in the document]
Comment (rayfajars) [reply src/components/layouts/ActionTable.vue]: thans for suggestion bli,

i am change [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/18/commits/a74b03f4b7fb12fb299da31a3c1e22ef3b9b7923)

File: src/components/RequestComponents/FormVerifikasiRequest.vue [hunk 2]
```diff
@@ -1,11 +1,22 @@
 <script>
+import HRCenter from "../layouts/HRCenter.vue";
+import { typeItemObjectOption } from "@/constants";
 export default {
+  components: { HRCenter },
   props: { conditionDetailVerifikasi: Object },
+  data() {
+    return {
+      typeItemObjectOption: typeItemObjectOption,
```
Comment (yoslie) [inline src/components/RequestComponents/FormVerifikasiRequest.vue:9]: if the key name same with the variable in value we can just write it
```
return {
  typeItemObjectOption
}
```
reff: https://eslint.org/docs/latest/rules/object-shorthand

File: src/components/RequestComponents/HeaderTable.vue [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
 export default {
   components: {
     LoadSvg,
   },
+  data() {
+    return {
+      divisiArrayOption: divisiArrayOption,
+      typeRequestObjectOption: typeRequestObjectOption,
+      statusObject: statusObject,
+      typeItemObjectOption: typeItemObjectOption,
```
Comment (yoslie) [inline src/components/RequestComponents/HeaderTable.vue:18]: this one too

Comment (yoslie) [reply src/components/RequestComponents/HeaderTable.vue:18]: are u not using `eslint` in this project?

File: src/components/RequestComponents/FormVerifikasiRequest.vue (hunk 2 above)
Comment (rayfajars) [reply src/components/RequestComponents/FormVerifikasiRequest.vue]: thanks for suggest kang yoslie,

i am change [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/18/commits/12f901485e31829058f2dfeabdc8c5a390b45b6b)

File: src/components/RequestComponents/HeaderTable.vue (hunk 3 above)
Comment (rayfajars) [reply src/components/RequestComponents/HeaderTable.vue]: thanks for suggest kang yoslie,

i am change [here](https://github.com/jabardigitalservice/digiteam-inventaris-app/pull/18/commits/12f901485e31829058f2dfeabdc8c5a390b45b6b)

**Review by yoslie — APPROVED** [review]

**Pull request merged**